### ISSUE TITLE
add an optionnal export flag to list-markets/list-pairs sub-command

### ISSUE THREAD

**Issue by gaugau3000** [freqtrade/freqtrade#2485]
**As** a trader
**I want** to export the pairlist generated by the   list-markets/list-pairs sub-command in a pairs.json file
**So that** i can automate the download  of the candles of new added market pairs (via the download-data command).

**Comment by xmatthias** [comment]
Odd question unrelated to the issue - why always the "as a trader" starting phrase?
it kinda reads odd - but maybe that's me or a language thing?

about the issue:
Does not work for you ... ? 
``` bash
freqtrade list-pairs --exchange binance --quote ETH --print-json > pairs.json
```  
from: https://www.freqtrade.io/en/latest/utils/#examples

The only point missint there is the redirect to `pairs.json` - but that's dangerous as it probably won't work in windows ...

**Comment by gaugau3000** [comment]
Yes it work i was thinking i will have the logs too with the > symbol (interesting : i have only test on linux). Many tks

The "As a" comes from the way to express features in [an agile way](https://www.mountaingoatsoftware.com/agile/user-stories).

Normally during the beginning of the project we define [personas](http://www.agilemodeling.com/artifacts/personas.htm) it is a way to deep into end users behave and be sure develop feature that fit well with the archetype.

For example in an app like freqtrade the personas could be have this type of profile:
-> the newbie who think he will find the holly grail comes from udemy (call him Eric).
-> an advanced programmer interested mostly in contribute in a well documented and tested project (call him Victor).
-> a financial profile who diversify with cryptocurrencies (call him Alexis).

and then use the As Eric I want ... so that ....

But you are true we don't have define personas or role :-) so i use the word trader as gereric user just to not break the user story like : i admit it is useless if i only use a generic type.

I was a scrum master in another life :-) so maybe i try to push agile practices.

**Comment by xmatthias** [comment]
> Yes it work i was thinking i will have the logs too with the > symbol (interesting : i have only test on linux).

This used to be the case, but was fixed in #2446  - changing logs to stderr instead of stdout, so forwarding output does work.

> But you are true we don't have define personas or role :-) so i use the word trader as gereric user just to not break the user story like : i admit it is useless if i only use a generic type.
> I was a scrum master in another life :-) so maybe i try to push agile practices.

Thanks for the extensive description, it now makes sense - please continue to do so.

**Comment by hroff-1902** [comment]
Both `>` (`1>`) and `2>` redirections work fine on Windows in cmd. The name of the null device is `NUL`, instead of `/dev/null`. that's the only difference.
